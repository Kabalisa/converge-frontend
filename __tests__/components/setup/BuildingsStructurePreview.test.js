import React from 'react';
import { shallow } from 'enzyme';
import BuildingsStructurePreview from '../../../src/components/onboarding/BuildingsSetup/BuildingsStructurePreview';

describe.only('Setup component', () => {
  const wrapper = shallow(<BuildingsStructurePreview />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
