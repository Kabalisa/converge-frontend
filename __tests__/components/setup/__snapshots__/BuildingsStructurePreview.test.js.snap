// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Setup component should render properly 1`] = `
<div
  className="rooms-preview"
>
  <div
    className="initial-preview-title"
  >
    Here is the preview of your setup
  </div>
  <img
    alt="initial preview when no data"
    className="initial-preview-image"
    src="test-file-stub"
  />
  <div
    className="initial-preview-bottom-text"
  >
    Please fill-in details on the left to continue
  </div>
</div>
`;
